README.md

```diff
@@ -1,12 +1,15 @@
 # Prometheus OpenStack exporter
 
+> [!NOTE]
+> This charm is under maintenance mode. Only critical bug will be handled.
```

Comment:
```suggestion
> This charm is under maintenance mode. Only critical bugs will be handled.
```
small nit

Reply:
Fixed. So I resolved this one by myself.